src/utils/tmdbAPI.ts

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+import { discoverRequestParams } from "../types";
+
+interface commonRequestParams {
+  api_key: string | undefined;
+  language?: string;
+}
+
+const commonParams: commonRequestParams = {
+  api_key: process.env.REACT_APP_TMDB_API_KEY,
```

Comment:
BTW, I suggest you to create `src/config.ts` where you will maintain all these environment variables.

Trust me, it's easier to take a look on a single file to answer "which environment variables do you use?" rather than to search them across all application.

```
export const TMDBAPIKey = process.env.REACT_APP_TMDB_API_KEY
```

Reply:
резонно